workflow/kfdrc_production_somatic_variant_cnv_wf.cwl

```diff
@@ -161,6 +161,19 @@ doc: |-
       - `Variant Effect Predictor`: kfdrc/vep:r93_v2
       - `Manta`: kfdrc/manta:latest
       - `bcftools` and `vcftools`: kfdrc/bvcftools:latest
+  1. For highly complex samples, some tools have shown themselves to require memory allocation adjustments:
+     Manta, GATK LearnReadOrientationModel, GATK GetPileupSummaries, and Vardict. Optional inputs exist to expand the
+     memory allocation for these jobs: manta_memory, learnorientation_memory, getpileup_memory, and vardict_ram,
+     respectively. For the java tools (Vardict and GATK), these values represent limits on the memory that can
+     be used for the respective jobs. Tasks will go to these values and not exceed it. They may succeed or fail,
+     but they will not exceed the limit established. The memory allocations for these is hardcapped. The memory
+     allocation option for Manta, conversely, is a soft cap. The memory requested will be allocated for the job
+     on a particular machine but once the task is running on the machine it may exceed that requested value. For example,
+     if Manta's memory allocation is set to 10 GB it will have 10 GB allocated to it at task creation, but, if the
+     task ends up running on a machine with more memory available, the task may use it. Setting a value here for Manta
+     will not prevent Manta from taking more than that value. The memory usage is Manta is limited by the machine hardware.
```

Comment:
Should this say "The memory usage in Manta is limited..."?

Reply:
fixed